src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ekaterina developer</title>
+  <script src="./formatDate.js"></script>
+</head>
+<body>
+  

```

Comment:
There is an unnecessary empty line before the closing body tag. It's a good practice to remove unnecessary empty lines to keep your code neat and readable.